packages/design-system-twrnc-preset/src/Theme/Theme.types.ts

```diff
@@ -1,25 +1,23 @@
 import type { create } from 'twrnc';
 
-import { ColorScheme } from '../twrnc-settings';
-import type { ColorSet } from '../twrnc-settings';
-
 export enum Theme {
-  Default = 'default',
-  Light = ColorScheme.Light,
-  Dark = ColorScheme.Dark,
+  Light = 'light',
+  Dark = 'dark',
 }
 
+export type ThemeType = Theme.Light | Theme.Dark;
+
+export type ColorSetListProps = {
+  [Theme.Light]: Record<string, string>;
+  [Theme.Dark]: Record<string, string>;
+};
+
 export type ThemeContextProps = {
   tw: ReturnType<typeof create>;
-  colorSet: ColorSet;
-  colorSetList: Record<string, string>;
-  theme: Theme;
-  setColorSet: (theme: ColorSet) => void;
-  setTheme: (scheme: Theme) => void;
+  theme: ThemeType;
 };
 
 export type ThemeProviderProps = {
   children: React.ReactNode;
-  colorSet?: ColorSet;
-  theme?: Theme;
+  theme: ThemeType;
 };
```

Comment:
Simplifying ThemeContext provider and types by removing `colorSet`, `colorSetList`, `setColorSet`, `setTheme`